src/config.ts

```diff
@@ -37,14 +37,65 @@ const clientConfig: ClientProps = {
     true
   ),
   enableLogging: envValueToBoolean(process.env.REACT_APP_OIDC_LOGGING, false),
-  tokenExchangePath: process.env.REACT_APP_OIDC_TOKEN_EXCHANGE_PATH
+  tokenExchangePath: process.env.REACT_APP_OIDC_TOKEN_EXCHANGE_PATH,
+  path: '/helsinkimvp',
+  routeMatch: '/helsinkimvp/:subpage?',
+  hasApiTokenSupport: true,
```

Comment:
Can `hasApiTokenSupport` be calculated from `tokenExchangePath` or `REACT_APP_*_TOKEN_EXCHANGE_PATH` (or some other setting) dynamically?

Reply:
Yes, no-one will probably set token path for Keycloak.